cli/src/helpers/git.ts

```diff
@@ -120,6 +120,7 @@ export const initializeGit = async (projectDir: string) => {
         cwd: projectDir,
       });
     }
+    await execa("git", ["add", "."], { cwd: projectDir });
     spinner.succeed(
       `${chalk.green("Successfully initialized")} ${chalk.green.bold("git")}\n`,
```

Comment:
maybe change this log to `initialised and staged` or something to let users know we did that?

Reply:
Sure, updated commit